### PR TITLE
sci-physics bumps

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: sci-libs/udunits/udunits-2.2.28.ebuild [hunk 1]
```diff
@@ -23,6 +23,10 @@ src_configure() {
 src_install() {
 	default
 
+	for i in udunits2 udunits2-{accepted,base,common,derived,prefixes}; do
```
Comment (SoapGentoo) [inline sci-libs/udunits/udunits-2.2.28.ebuild:26]: declare `local i` before

File: sci-physics/lhapdf/lhapdf-6.3.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+S="${WORKDIR}/${MY_PF}"
+
+src_configure() {
+	econf \
+		--disable-static \
+		$(use_enable python)
+
+	if use python; then
+		cd "${S}/wrappers/python" && distutils-r1_src_prepare
```
Comment (SoapGentoo) [inline sci-physics/lhapdf/lhapdf-6.3.0.ebuild:45]: separate this into 2 statements

File: sci-physics/lhapdf/lhapdf-6.3.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+	fi
+}
+
+src_test() {
+	emake -C tests
+}
+
+src_install() {
+	emake DESTDIR="${D}" install
+	use doc && dodoc -r doc/doxygen/*
```
Comment (SoapGentoo) [inline sci-physics/lhapdf/lhapdf-6.3.0.ebuild:63]: `use doc && dodoc -r doc/doxygen/.` (avoid globs)

File: sci-physics/yoda/yoda-1.8.5.ebuild [hunk 4]
```diff
@@ -34,6 +33,7 @@ src_test() {
 
 src_install() {
 	default
-	newbashcomp "${ED%/}"/usr/share/YODA/yoda-completion ${PN}
-	rm "${ED%/}"/usr/share/YODA/yoda-completion || die
+	newbashcomp "${ED}"/usr/share/YODA/yoda-completion ${PN}
```
Comment (SoapGentoo) [inline sci-physics/yoda/yoda-1.8.5.ebuild:36]: inline `${PN}` here

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: sci-libs/udunits/udunits-2.2.28.ebuild (hunk 1 above)
Comment (jsmolic) [reply sci-libs/udunits/udunits-2.2.28.ebuild:27]: done

File: sci-physics/lhapdf/lhapdf-6.3.0.ebuild (hunk 2 above)
Comment (jsmolic) [reply sci-physics/lhapdf/lhapdf-6.3.0.ebuild]: done

File: sci-physics/lhapdf/lhapdf-6.3.0.ebuild (hunk 3 above)
Comment (jsmolic) [reply sci-physics/lhapdf/lhapdf-6.3.0.ebuild]: done

File: sci-physics/yoda/yoda-1.8.5.ebuild (hunk 4 above)
Comment (jsmolic) [reply sci-physics/yoda/yoda-1.8.5.ebuild]: done

File: sci-physics/lhapdf/lhapdf-6.3.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+S="${WORKDIR}/${MY_PF}"
+
+src_configure() {
+	econf \
+		--disable-static \
+		$(use_enable python)
+
+	if use python; then
+		cd "${S}/wrappers/python" || die
```
Comment (SoapGentoo) [inline sci-physics/lhapdf/lhapdf-6.3.0.ebuild:45]: quoting?

File: sci-physics/lhapdf/lhapdf-6.3.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+		distutils-r1_src_prepare
+	fi
+}
+
+src_compile() {
+	emake all $(use doc && echo doxy)
+
+	if use python; then
+	   cd "${S}/wrappers/python" || die
+	   distutils-r1_src_compile
```
Comment (SoapGentoo) [inline sci-physics/lhapdf/lhapdf-6.3.0.ebuild:55]: wrogn indentation

File: sci-physics/lhapdf/lhapdf-6.3.0.ebuild [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+	emake -C tests
+}
+
+src_install() {
+	emake DESTDIR="${D}" install
+	use doc && dodoc -r doc/doxygen/.
+	use examples && doins examples/*.cc
+
+	if use python; then
+		cd "${S}/wrappers/python" || die
```
Comment (SoapGentoo) [inline sci-physics/lhapdf/lhapdf-6.3.0.ebuild:69]: ditto

File: sci-physics/pythia/pythia-6.4.28-r2.ebuild [hunk 8]
```diff
@@ -27,11 +27,15 @@ SRC_URI="
...
+IUSE="doc examples"
+
+PATCHES=( "${FILESDIR}"/${P}-fno-common.patch )
 
 src_prepare() {
 	cp ../pythia6/tpythia6_called_from_cc.F .
 	cp ../pythia6/pythia6_common_address.c .
+	default
+
 	cat > configure.ac <<-EOF
```
Comment (SoapGentoo) [inline sci-physics/pythia/pythia-6.4.28-r2.ebuild:39]: missing `|| die`

File: sci-physics/pythia/pythia-6.4.28-r2.ebuild [hunk 9]
```diff
@@ -50,11 +54,12 @@ src_prepare() {
 	done
 	echo  >> Makefile.am "  ssmssm.f sugra.f visaje.f pdfset.f \\"
 	echo  >> Makefile.am "  tpythia6_called_from_cc.F pythia6_common_address.c"
```
Comment (SoapGentoo) [inline sci-physics/pythia/pythia-6.4.28-r2.ebuild:56]: all the echo's are missing `|| die`s

File: sci-physics/pythia/pythia-8.2.26.ebuild [hunk 10]
```diff
@@ -84,18 +89,15 @@ src_prepare() {
...
-	eapply "${FILESDIR}/${PN}8209-root-noninteractive.patch"
-	eapply_user
+	sed -i \
+		-e '/TARGETS=$(LOCAL_LIB)\/libpythia8\.a/d' \
+		-e 's:libpythia8\.a$:libpythia8$(LIB_SUFFIX):g' \
+		Makefile || die
+	sed -i 's:libpythia8\.a:libpythia8$(LIB_SUFFIX):g' \
+		examples/Makefile || die
+
+	default
```
Comment (SoapGentoo) [inline sci-physics/pythia/pythia-8.2.26.ebuild:100]: prefer to patch before you `sed`

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: sci-physics/lhapdf/lhapdf-6.3.0.ebuild (hunk 5 above)
Comment (jsmolic) [reply sci-physics/lhapdf/lhapdf-6.3.0.ebuild]: sure, fixed ;)

File: sci-physics/lhapdf/lhapdf-6.3.0.ebuild (hunk 6 above)
Comment (jsmolic) [reply sci-physics/lhapdf/lhapdf-6.3.0.ebuild]: done

File: sci-physics/lhapdf/lhapdf-6.3.0.ebuild (hunk 7 above)
Comment (jsmolic) [reply sci-physics/lhapdf/lhapdf-6.3.0.ebuild]: done

File: sci-physics/pythia/pythia-6.4.28-r2.ebuild (hunk 8 above)
Comment (jsmolic) [reply sci-physics/pythia/pythia-6.4.28-r2.ebuild]: done

File: sci-physics/pythia/pythia-6.4.28-r2.ebuild (hunk 9 above)
Comment (jsmolic) [reply sci-physics/pythia/pythia-6.4.28-r2.ebuild]: done

File: sci-physics/pythia/pythia-8.2.26.ebuild (hunk 10 above)
Comment (jsmolic) [reply sci-physics/pythia/pythia-8.2.26.ebuild]: done

File: sci-physics/pythia/pythia-6.4.28-r2.ebuild [hunk 11]
```diff
@@ -41,28 +45,31 @@ src_prepare() {
...
 }
 
 src_install() {
 	default
 	dodoc update_notes.txt
 	use doc && dodoc "${DISTDIR}"/lutp${DOC_PV}man2.pdf
 	if use examples; then
-		insinto /usr/share/doc/${PF}
+		insinto /usr/share/${PN}
 		doins -r "${WORKDIR}"/examples
```
Comment (SoapGentoo) [inline sci-physics/pythia/pythia-6.4.28-r2.ebuild:71]: changes these 2 lines:
```
dodoc -r "${WORKDIR}"/examples
docompress -x /usr/share/doc/${PF}/examples
```

File: sci-physics/pythia/pythia-8.2.26-r1.ebuild [hunk 12]
```diff
@@ -157,25 +159,25 @@ src_install() {
...
 	dobin bin/pythia8-config
 	doheader -r include/*
-	dolib lib/*
+	dolib.so lib/libpythia8.so
 	insinto "${PYTHIADIR}"
 	doins -r share/Pythia8/xmldoc examples/Makefile.inc
 
-	echo "PYTHIA8DATA=${EPYTHIADIR}/xmldoc" >> 99pythia8
+	echo "PYTHIA8DATA=${EPYTHIADIR}/xmldoc" >> 99pythia8 || die
 	doenvd 99pythia8
```
Comment (SoapGentoo) [inline sci-physics/pythia/pythia-8.2.26-r1.ebuild:167]: Replace these two lines by
```
newenvd - 99pythia8 <<- _EOF_
	PYTHIA8DATA=${EPYTHIADIR}/xmldoc
_EOF_
```

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: sci-physics/pythia/pythia-6.4.28-r2.ebuild (hunk 11 above)
Comment (jsmolic) [reply sci-physics/pythia/pythia-6.4.28-r2.ebuild]: Done

File: sci-physics/pythia/pythia-8.2.26-r1.ebuild (hunk 12 above)
Comment (jsmolic) [reply sci-physics/pythia/pythia-8.2.26-r1.ebuild]: Thanks, done

**Pull request closed without merging**